spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/TaskExecutionController.java

```diff
@@ -97,7 +97,7 @@ public TaskExecutionController(TaskExplorer explorer, TaskService taskService,
 	@ResponseStatus(HttpStatus.OK)
 	public PagedResources<TaskExecutionResource> list(Pageable pageable,
 			PagedResourcesAssembler<TaskJobExecutionRel> assembler) {
-		Page<TaskExecution> taskExecutions = explorer.findAll(pageable);
+		Page<TaskExecution> taskExecutions = this.taskService.findAll(pageable);
```

Comment:
For the record, I'm not a fan of creating a custom extension of the `TaskExplorer` for this kind of work.  This functionality doesn't have anything to do with the `TaskExplorer` itself.  It is related to how SCDF displays data.  That's an SCDF concern and by extending `TaskExplorer` you create unneeded coupling to the `TaskExplorer`.  To be honest, I view this as a formatting issue that should be handled in the controller but the service is fine as well IMHO.